### PR TITLE
Added the two blog posts that accompany the SPI Playgrounds app launch.

### PR DESCRIPTION
**Pull request by daveverwer**

### CODE REVIEW

File: Content/posts/how-the-swift-package-index-project-got-started.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+description: How did the Swift Package Index project get started, and why does a button feature so prominently in the story? Read on to find out.
+---
+
+It’s been [one year since we made the first commit](https://twitter.com/_sa_s/status/1386033811348197380) on the [Swift Package Index repository](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server), and we think that deserves a little celebration!
+
+<center style="font-size: 30px;">🎂</center>
+
+Did you enjoy the cake? 😂 No? Well, how about we also tell you the story of how this project started?
+
+**Note:** We usually write blog posts here in the third person as this project is very much a joint effort, but this is a story with lots of references to both [Dave](https://twitter.com/daveverwer) and [Sven](https://twitter.com/_sa_s) individually, so I’ll write it in the first person. _Dave_
```
Comment (finestructure) [inline Content/posts/how-the-swift-package-index-project-got-started.md:13]: I think you meant `first person plural` as our usual perspective?

Comment (daveverwer) [reply Content/posts/how-the-swift-package-index-project-got-started.md:13]: Fixed.

File: Content/posts/how-the-swift-package-index-project-got-started.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+I’m delighted to say that today sees the launch of “the button”! But what does the button do? Read more about that in the [official launch post](/posts/launching-the-swift-package-index-playgrounds-app-for-macos).
+
+That deserves more than a cake! 🥂🍾
+
+I’m happy that we **finally** got to bring Sven’s original idea to fruition through this project, but this celebration is about much more than that! Working with Sven on this project has been a pleasure from the beginning. We work incredibly well together and are making something that feels significant and important. The button is part of that, but the potential of the site is so much more.
+
+It’s also worth mentioning that if you want to hear Sven and I talk about this story in person, we also [discussed it on John Sundell’s podcast](https://swiftbysundell.com/podcast/75/) last year.
```
Comment (finestructure) [inline Content/posts/how-the-swift-package-index-project-got-started.md:35]: Nit: `also` ... `also`

Comment (daveverwer) [reply Content/posts/how-the-swift-package-index-project-got-started.md:35]: Yep, good catch.

**Review by finestructure — APPROVED** [review]

**Pull request merged**